src/Raven.Server/RavenServer.cs

```diff
@@ -152,10 +151,17 @@ void ConfigureKestrel(KestrelServerOptions options)
                     }
                 }
 
+                var addresses = new List<string>();
+                var serverUri = new Uri(Configuration.Core.ServerUrl);
+                foreach (var address in GetListenIpAddresses(serverUri.DnsSafeHost).Where(a=>a.IsIPv6LinkLocal == false))
```

Comment:
You are right, this is redundant. We already doing this inside the *GetListenIpAddresses* function